`NugetITCase` with `TestDeployment`

olenagerasimova opened the pull request:
Part of #855
Remade `NugetITCase` to use `TestDeployment`

genryxy commented:
@olenagerasimova please check failure with `.pdd`

genryxy commented on src/test/java/com/artipie/nuget/NugetITCase.java:
```diff
@@ -23,120 +23,43 @@
-        );
-    }
-
-    @Test
-    @Timeout(30)
-    @Disabled
-    void shouldInstallPushedPackage() throws Exception {
```
@olenagerasimova also one concern about this test case. Probably remain `@todo` to add installation test or rewrite it

olenagerasimova commented:
@genryxy thanks, the test method both push and installs the package, I've corrected method name and do not think another test is required here. Also, removed unused class and corrected PDD config.

genryxy reviewed (approved):
@olenagerasimova thanks, got it
